apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/navigation-menu/index.php

```diff
@@ -16,9 +16,11 @@
 function render_navigation_menu_block( $attributes ) {
 	$styles = '';
 
-	$class = 'menu-primary-container';
```

Comment:
I'm getting some errors/warnings related to this variable in the page editor:

<img width="1237" alt="Screen Shot 2019-10-29 at 12 46 50 PM" src="https://user-images.githubusercontent.com/6265975/67803393-30842000-fa4a-11e9-8e9f-16d5dd590c30.png">

<img width="811" alt="Screen Shot 2019-10-29 at 12 47 05 PM" src="https://user-images.githubusercontent.com/6265975/67803425-3b3eb500-fa4a-11e9-8650-1055acaa5fe5.png">

Reply:
Good catch!
(I'm very unimpressed by my PHPCS for not catching it earlier, though 😤)